[1주차] 구현/창희/S5_2563.java

```diff
@@ -0,0 +1,31 @@
+package backjoon;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.StringTokenizer;
+
+public class S5_2563 {
+    public static void main(String[] args) throws NumberFormatException, IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        int[][] canvas = new int[100][100];
+
+        int n = Integer.parseInt(br.readLine());
+        int result = 0;
+        for(int i=0; i<n; i++) {
+            StringTokenizer st = new StringTokenizer(br.readLine());
+            int x = Integer.parseInt(st.nextToken());
+            int y = Integer.parseInt(st.nextToken());
+            for(int j=0; j < 10; j++) {
+                for(int k=0; k<10; k++) {
+                    canvas[x+j][y+k]++;
+                    if(canvas[x+j][y+k] == 1) {
+                        result++;
+                    }
+
+                }
+            }
+        }
+        System.out.println(result);
+    }
+}
```

Comment:
import java.util.Scanner;

public class Main{
    private static int[][] arr = new int[100][100];

    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        int paintCount = 0;

        for(int i = 0; i<n; i++){
            int x = sc.nextInt();
            int y = sc.nextInt();

            for(int j = x; j<x+10; j++){
                for(int q=y; q<y+10;q++){
                    if(arr[j-1][q-1] == 0){
                        arr[j-1][q-1]++;
                        paintCount++;
                    }
                }
            }
        }
        System.out.println(paintCount);
    }
}